examples/react/script.js

```diff
@@ -11,6 +13,33 @@ const getPosition = (elapsedTime, h, k) => {
   return ypos;
 };
 
+// performs a Quadratic Ease in and Ease out repeatedly
+function useQuadBounce({
+  duration = 1150,
+  start = 0,
+  end = 160,
+} = {}) {
+  const timeStart = useRef(Date.now());
+  const interval = useRef();
+  const [value, setValue] = useState(start);
+
+  const updateValue = useCallback(() => {
+    const time = Date.now() - timeStart.current;
+    setValue(start + getPosition(time, duration / 2, end - start));
+  }, [duration, start, end]);
+
+  useEffect(() => {
+    interval.current = setInterval(updateValue, 20);
```

Comment:
I just realized that intervals of 20ms makes this run at 50 frames per second (1000 ms/sec ÷ 20 ms/frame = 50 frame/sec). Could we change this to 16ms?